api/helpers/things.js

```diff
@@ -219,21 +235,25 @@ function getEditXById(type) {
             }
           } else if (
             // fields that are lists of strings
-            [
-              "tags",
-              "links",
-              "images",
-              "videos",
-              "files",
-              "if_voting",
-              "evaluation_reports",
-              "evaluation_links"
-            ].includes(key)
+            ["tags", "if_voting"].includes(key)
           ) {
             updatedThingFields.push({
               key: as.name(key),
               value: as.strings(value)
             });
+          } else if (
+            // fields that are rich links
+            ["links", "images", "videos", "evaluation_links"].includes(key)
```

Comment:
this list should include `audio` and should not include `images`.

Reply:
I think only Organizations still uses this and the whole function will go away once I update Organizations (today).